serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,286 @@
-= Create your first workflow service
\ No newline at end of file
+= Create your first workflow service
+
+Learn how to create a Hello World Serverless Workflow application. This guide covers:
+
+* Bootstrapping an application
+* Creating a workflow
+* Running the application
+* Testing the application
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* Roughly 10 minutes
+* https://adoptium.net/[JDK 11+] installed with `JAVA_HOME` configured appropriately
+* https://maven.apache.org/install.html[Apache Maven] {required_maven_version}
+* https://quarkus.io/guides/cli-tooling[Quarkus CLI] (Optional)
+
+To edit your workflows:
+
+* Visual Studio Code with https://marketplace.visualstudio.com/items?itemName=redhat.java[Red Hat Java Extension]
+and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-serverless-workflow-editor[Red Hat Serverless Workflow Editor]
+installed (Optional)
+
+It's recommended that you take a look at
+link:getting-familiar-with-our-tooling.html[this guide] to get familiar with our tooling and the required dependencies.
```

Comment:
@hbelmiro I think we can do this alternatively:

```suggestion
link:getting-familiar-with-our-tooling.html[this guide] to get familiar with our tooling and the required dependencies.
```
```suggestion

xref:getting-started/getting-familiar-with-our-tooling.adoc[this guide] to get familiar with our tooling and the required dependencies.
```

@akumar074 can you confirm?